osxcollector/output_filters/opendns/related_domains.py

```diff
@@ -10,14 +21,28 @@
 from osxcollector.output_filters.util.domains import expand_domain
 
 
+DEFAULT_RELATED_DOMAINS_GENERATIONS = 2
+
+
 class RelatedDomainsFilter(OutputFilter):
 
-    """Uses OpenDNS to find domains related to input domains or ips.
+    """Uses OpenDNS to find domains related to input domains or IPs.
 
     A whitelist of domains to ignore is read during initialization.
+    Adds 'osxcollector_related' key to the output:
+    ```python
+    {
+       'osxcollector_related': {
+           'domains': {
+               'domain_in_line.com': ['related_domain.com'],
+               'another.com': ['1.2.3.4']
+           }
+        }
+    }
+    ```
     """
 
-    def __init__(self, initial_domains=None, initial_ips=None, generations=2, related_when=None):
+    def __init__(self, initial_domains=None, initial_ips=None, generations=DEFAULT_RELATED_DOMAINS_GENERATIONS, related_when=None):
```

Comment:
Line length is declared and enforced to [140 chars in tox.ini](https://github.com/Yelp/osxcollector/blob/master/tox.ini#L19). This fits.